### PR TITLE
Updated the documentation of the API endpoints and there description

### PR DESCRIPTION
**Pull request by AKMaily**

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -304,28 +263,16 @@
...
-                  "type": "number",
-                  "format": "float"
-                },
-                "amplitude": {
-                  "type": "number",
-                  "format": "float"
-                }
-              }
-            }
+          "meta": {
```
Comment (dobe-1) [inline openapi.json]: I think it is not necessary to define a `meta` object here. The description and the properties can be defined directly in the `result` object. If you have a different opinion please correct me.

File: openapi.json [hunk 2]
```diff
@@ -272,12 +231,12 @@
...
-            "type": "array",
-            "items": {
-              "type": "string"
-            }
+            "type": "object",
+            "description": "Here are all informations that are needed for the formular fields",
+            "properties": {}
           },
           "data": {
+            "title": "Daten",
```
Comment (dobe-1) [inline openapi.json]: This line can be removed

**Review by dobe-1 — CHANGES_REQUESTED** [review]

File: openapi.json (hunk 1 above)
Comment (AKMaily) [reply openapi.json]: So the correct order would be 
Result: { 
    type: object ,  
   description, 
    properties: { }}} ?

Comment (dobe-1) [reply openapi.json]: Yes

**Pull request merged**